uaclient/defaults.py

```diff
@@ -10,6 +10,8 @@
 DEFAULT_MACHINE_TOKEN_PATH = DEFAULT_DATA_DIR + "/private/machine-token.json"
 DEFAULT_CONFIG_FILE = UAC_ETC_PATH + "uaclient.conf"
 DEFAULT_HELP_FILE = UAC_ETC_PATH + "help_data.yaml"
+PRIVATE_MACHINE_TOKEN_FILE = "machine-token.json"
+PUBLIC_MACHINE_TOKEN_FILE = "pub-machine-token.json"
```

Comment:
Bikeshedding opportunity: Do they need separate names? Or is it sufficient that one is in the `private/` directory?

Reply:
The final name was still to be determined, your suggestion is a good one.